src/pages/networks/NetworkPeers.tsx

```diff
@@ -0,0 +1,149 @@
+import ResourceLink from "components/ResourceLink";
+
+interface Props {
+  network: LxdNetwork;
+  project: string;
+}
+
+const NetworkPeers: FC<Props> = ({ network, project }) => {
+  const docBaseLink = useDocs();
+  const notify = useNotify();
+
+  const {
+    data: peers = [],
+    error,
+    isLoading,
+  } = useQuery({
+    queryKey: [
+      queryKeys.projects,
+      project,
+      queryKeys.networks,
+      network.name,
+      queryKeys.peers,
+    ],
+    queryFn: async () => fetchNetworkPeers(network.name, project),
+  });
+
+  if (error) {
+    notify.failure("Loading local network peers failed", error);
+  }
+
+  const hasNetworkPeers = peers.length > 0;
+
+  const headers = [
+    { content: "Name", sortKey: "name" },
+    { content: "Target Network", sortKey: "targetnetwork" },
+    { content: "Description", sortKey: "description" },
+    { content: "Project", sortKey: "project" },
+    { content: "Status", sortKey: "status" },
+  ];
+
+  const rows = peers.map((peer) => {
+    return {
+      key: peer.name,
+      columns: [
+        {
+          content: peer.name,
+          role: "cell",
+          "aria-label": "Name",
+        },
+        {
+          content: peer.target_network,
+          role: "rowheader",
+          "aria-label": "Target network",
+        },
+        {
+          content: peer.description,
+          role: "cell",
+          "aria-label": "Description",
+        },
+        {
+          content: peer.project && (
+            <ResourceLink
+              type="project"
+              value={peer.project}
+              to={`/ui/project/${encodeURIComponent(peer.project)}`}
+            />
+          ),
+          role: "cell",
+          "aria-label": "Project",
+        },
+        {
+          content: peer.status,
+          role: "cell",
+          "aria-label": "Status",
+        },
+      ],
+      sortData: {
+        hostname: peer.name.toLowerCase(),
+        targetnetwork: peer.target_network,
+        description: peer.description,
+        project: peer.project?.toLowerCase(),
+        status: peer.status?.toLowerCase(),
+      },
+    };
+  });
+
+  if (isLoading) {
+    return <Spinner className="u-loader" text="Loading..." isMainComponent />;
+  }
+
+  return (
+    <Row>
+      {hasNetworkPeers && (
+        <ScrollableTable
+          dependencies={peers}
+          tableId="network-peer-table"
+          belowIds={["status-bar"]}
+        >
+          <MainTable
+            id="network-peer-table"
+            headers={headers}
+            expanding
+            rows={rows}
+            responsive
+            sortable
+            className="u-table-layout--auto"
+            emptyStateMsg="No data to display"
+          />
+        </ScrollableTable>
+      )}
+      {!isLoading && !hasNetworkPeers && (
+        <EmptyState
+          className="empty-state"
+          image={<Icon className="empty-state-icon" name="exposed" />}
+          title="No network peers found"
```

Comment:
```suggestion
          title="No local peerings found"
```
More consistent labels.